apps/docs/src/app/core/component-docs/message-strip/message-strip-docs.component.html

```diff
@@ -0,0 +1,42 @@
+<fd-docs-section-title [id]="'message-strip-types'" [componentName]="'messageStrip'">
+    Message Strip Types
+</fd-docs-section-title>
+<description>A number of message strip types are available. They offer preset styling for different contextual situations. Can be one of 'warning', 'success', 'information', 'error' or normal (default). </description>
+<component-example>
+    <fd-message-strip-example></fd-message-strip-example>
+</component-example>
+<code-example [exampleFiles]="messageStripBasicExample"></code-example>
+
+<separator></separator>
+
+<fd-docs-section-title [id]="'message-strip-types'" [componentName]="'messageStrip'">
```

Comment:
Repeating `[id]` value, also for "Message Strips With Customizable Width"

Reply:
Fixed